packages/source-analysis/src/routeTrees/extractRouteTree.test.ts

```diff
@@ -34,10 +39,17 @@ test.each([
   {
     name: "hono factory",
     location: "../../test-cases/hono-factory",
+    expectedErrorCount: 0,
   },
   {
     name: "import as",
     location: "../../test-cases/import-as",
+    expectedErrorCount: 0,
+  },
+  {
+    name: "zod-openapi",
+    location: "../../test-cases/zod-openapi",
+    expectedErrorCount: 2,
```

Comment:
unclear to me why the expectedErrorCount is 2 here! could you add a comment maybe?